### PR TITLE
Fixed Save to pdf and added new features

### PR DESCRIPTION
**Pull request by MaciejDengusiak**
### Issues addressed by this PR

Fixes #

### CODE REVIEW

File: Mollier/SAM.Core.Mollier.UI/SAM.Core.Mollier.UI.csproj [hunk 1]
```diff
@@ -32,6 +32,9 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Aspose.Imaging">
```
Comment (ZiolkowskiJakub) [inline Mollier/SAM.Core.Mollier.UI/SAM.Core.Mollier.UI.csproj:35]: Do we need that?

File: Mollier/SAM.Core.Mollier.UI/SAM.Core.Mollier.UI.csproj [hunk 2]
```diff
@@ -32,6 +32,9 @@
...
+      <HintPath>F:\EMFtoPDF\net5.0\Aspose.Imaging.dll</HintPath>
```
Comment (ZiolkowskiJakub) [inline Mollier/SAM.Core.Mollier.UI/SAM.Core.Mollier.UI.csproj:36]: Do NOT use local absolute path in this case

File: Mollier/SAM.Core.Mollier.UI/Temporary/ExportPDF.cs [hunk 3]
```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Windows.Forms;
+using System.Windows.Forms.DataVisualization.Charting;
+using Aspose.Imaging;
```
Comment (ZiolkowskiJakub) [inline Mollier/SAM.Core.Mollier.UI/Temporary/ExportPDF.cs:4]: Do we need that?

**Review by ZiolkowskiJakub — CHANGES_REQUESTED** [review]

File: Mollier/SAM.Core.Mollier.UI/Temporary/ExportPDF.cs (hunk 3 above)
Comment (MaciejDengusiak) [reply Mollier/SAM.Core.Mollier.UI/Temporary/ExportPDF.cs:4]: DONE

**Review by ZiolkowskiJakub — APPROVED** [review]

**Pull request merged**